state: default

# CV information
cv_server: /CV_connect/req_cv
pull_image_size:
  width: 1280
  height: 736

# ROS information
info_topic:  /zed2i/zed_node/rgb/camera_info
image_topic: /zed2i/zed_node/rgb/image_rect_color
depth_topic: /zed2i/zed_node/depth/depth_registered
pcl2_topic:  /zed2i/zed_node/point_cloud/cloud_registered